Assets/_BForBoss/_Character/Scripts/PlayerSlowMotionBehaviour.cs

```diff
@@ -26,21 +23,14 @@ private void Start()
         {
             _fixedDeltaTime = Time.fixedDeltaTime;
         }
-
-        public void SetupPlayerInput(InputAction slowMoInput)
```

Comment:
Removed the `InputAction` from SlowTime, and handle everything in `PlayerMovement` script